h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -1,9 +1,15 @@
 'use strict';
 
+var escapeHtml = require('escape-html');
```

Comment:
lol I did that and then removed it thinking you would say not to have two lines :P